packages/shared/lib/services/sync/run.service.ts

```diff
@@ -219,411 +212,274 @@ export default class SyncRun {
-            } else if (this.isPostConnectionScript) {
-                syncData = {
-                    runs: 'every 5 minutes',
-                    returns: [],
-                    track_deletes: false,
-                    is_public: false,
-                    fileLocation: this.fileLocation
-                } as NangoIntegrationData;
+        if (this.debug) {
+            const content = `Last sync date is ${lastSyncDate}`;
+            if (this.writeToDb) {
+                await createActivityLogMessage({
+                    level: 'debug',
+                    environment_id: this.nangoConnection.environment_id,
+                    activity_log_id: this.activityLogId as number,
+                    timestamp: Date.now(),
+                    content
+                });
+                await this.logCtx?.debug(content);
             } else {
-                syncData = (syncObject['syncs'] ? syncObject['syncs'][this.syncName] : syncObject[this.syncName]) as unknown as NangoIntegrationData;
+                logger.info(content);
             }
+        }
 
-            const { returns: models, track_deletes: trackDeletes, is_public: isPublic } = syncData;
-
-            if (syncData.sync_config_id) {
-                if (this.debug) {
-                    const content = `Sync config id is ${syncData.sync_config_id}`;
-                    if (this.writeToDb) {
-                        await createActivityLogMessage({
-                            level: 'debug',
-                            environment_id: this.nangoConnection.environment_id,
-                            activity_log_id: this.activityLogId as number,
-                            timestamp: Date.now(),
-                            content
-                        });
-                        await this.logCtx?.debug(content);
-                    } else {
-                        logger.info(content);
-                    }
-                }
+        const startTime = Date.now();
+        const syncStartDate = new Date();
+        try {
+            result = true;
 
-                if (this.syncJobId) {
-                    await addSyncConfigToJob(this.syncJobId, syncData.sync_config_id);
-                }
+            if (typeof nangoProps.accountId === 'number') {
+                metrics.increment(getMetricType(this.determineExecutionType()), 1, { accountId: nangoProps.accountId });
             }
 
-            if (!isCloud && !integrationFilesAreRemote && !isPublic) {
-                const { path: integrationFilePath, result: integrationFileResult } = localFileService.checkForIntegrationDistFile(
-                    this.syncName,
-                    providerConfigKey,
-                    this.loadLocation
-                );
-                if (!integrationFileResult) {
-                    const message = `Integration was attempted to run for ${this.syncName} but no integration file was found at ${integrationFilePath}.`;
```

Comment:
Removed this check for local integration, this check is already done by the CLI and by local-integration, it's one less thing to refactor/carry in this class but let me know if you agree